examples/scripts/experiment_drive_cycle.py

```diff
@@ -0,0 +1,92 @@
+#
+# Constant-current constant-voltage charge with US06 Drive Cycle using Experiment Class.
+#
+import pybamm
+import numpy as np
+import pandas as pd
+import os
+
+os.chdir(pybamm.__path__[0] + "/..")
+
+pybamm.set_logging_level("INFO")
+
+# import drive cycle from file
+drive_cycle_current = pd.read_csv(
+    "pybamm/input/drive_cycles/US06.csv", comment="#", header=None
+).to_numpy()
+
+
+# Map Drive Cycle
+def Map_Drive_Cycle(x, min_ip_value, max_ip_value, min_op_value, max_op_value):
```

Comment:
functions should use `lower_case`

Reply:
Done

Reply:
I removed the voltage drive cycle as this specific example didn't work